tests/testImageDecorrelation.py

```diff
@@ -84,17 +86,17 @@ def makeFakeImages(xim=None, yim=None, sig1=0.2, sig2=0.2, psf1=2.2, psf2=3.3, o
     """
     np.random.seed(seed)
 
-    psf1 = [2.2, 2.2] if psf1 is None else psf1
+    psf1 = [3.3, 3.3] if psf1 is None else psf1
```

Comment:
Thanks for this. 
I'll go with this discussion: https://stackoverflow.com/a/19773559
Keeping the `hasattr` call and adding the `basestring` check.